project/Dependencies.scala

```diff
@@ -36,6 +36,7 @@ object Dependencies {
 
   val apacheIO            = "commons-io"                  % "commons-io"               % "2.6"
   val apacheMath          = "org.apache.commons"          % "commons-math3"            % "3.6.1"
+  val apacheValidator     = "commons-validator"           % "commons-validator"        % "1.4.0"
```

Comment:
I'm curious about what the rationale for this lib is - it looks kind of gross and java-y to me and will require CQ. I'm not saying 'no' to it necessarily, but it does seem heavier than necessary for this PR

Reply:
My main reason for adding it was to provide an easy way to validate a `URL`. Though, now that I think on it, I guess I could just catch the error produced by `URL` instead.